packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -650,15 +651,27 @@ export default class TradeStore extends BaseStore {
         }
     }
 
+    @action.bound
+    initAccountCurrency(new_currency) {
+        runInAction(async() => {
```

Comment:
Can the `initAccountCurrency` function become async itself and then we could remove the `runInAction`?

Reply:
Sure.